sheepdog/auth/__init__.py

```diff
@@ -19,6 +20,7 @@
 
 logger = get_logger(__name__)
 
+AUTHZ_CACHE = SimpleCache(default_timeout=1)
```

Comment:
the PR lgtm - we could add quick unit tests to be extra thorough since it's authz stuff - make sure the cache expires after 1s, and make sure requests from different users are cached separately

Reply:
I've added some unit tests to test the cache.